cli/packages/prisma-client-lib/src/codegen/typescript-client.ts

```diff
@@ -771,7 +771,7 @@ ${fieldDefinition}
         ? `export type ${typeName} = AtLeastOne<{
         ${fieldDefinition.replace('?:', ':')}
       }>`
-        : `export interface ${typeName}${typeName === 'Node' ? 'Node' : ''}${
```

Comment:
`${typeName === 'Node' ? 'Node' : ''}` [was added here](https://github.com/prisma/prisma/commit/5de11f305c0a77d6ff9daf028aba0a68ddd87d58#diff-d1ac2c5edb46cae3a12f4ed0eef31d66), and maybe should have been removed [in this commit](https://github.com/prisma/prisma/commit/7602bfef7062eb7bfc3ec13cae9227571b57168f).